crates/risc0-boundless/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
```

Comment:
Sorry but i am very opinionated about introducing crates. Especially this when we have `risc0-bonsai` and `risc0-boundless` where both can live in a single `proving-clients` crate or something like that.